packages/ember-glimmer/lib/resolver.ts

```diff
@@ -353,4 +369,16 @@ export default class RuntimeResolver implements IRuntimeResolver<OwnedTemplateMe
 
     return definition;
   }
+
+  _lookupComponentManager(owner: Owner, managerId: string): CustomComponentManager {
+    if (this.customManagerCache.has(managerId)) {
+      return this.customManagerCache.get(managerId)!;
+    }
+    let delegate = owner.lookup<ManagerDelegate>(`component-manager:${managerId}`);
+    let manager = new CustomComponentManager(delegate);
```

Comment:
Lets also pass the `owner` to the constructor here. It will allow us to more easily participate in the renderer's view registry when we want to make it work for the Ember Inspector.